precompiles/proxy/src/lib.rs

```diff
@@ -197,7 +197,12 @@ where
 		}
 		.into();
 
-		<RuntimeHelper<Runtime>>::try_dispatch(handle, Some(origin).into(), call)?;
+		<RuntimeHelper<Runtime>>::try_dispatch(
+			handle,
+			Some(origin).into(),
+			call,
+			SYSTEM_ACCOUNT_SIZE,
```

Comment:
```suggestion
			0,
```
add_proxy can't create a new account

Reply:
My bad on this one, I thought it might add an account if the proxy account doesn't exist (like to have a provider or kind of reference)